src/bluetoothdevice.h

```diff
@@ -145,7 +165,15 @@ class bluetoothdevice : public QObject {
     metric m_watt;
     metric WattKg;
     metric WeightLoss;
+
+    /**
+     * @brief The speed at which the crank is turning. Units: revolutions per minute (RPM)
+     */
```

Comment:
Perhaps BluetoothDevice::lastCrankEventTime should be moved to class bike, or renamed to something more generic, e.g. lastActionEventTime.

Reply:
i'm using that name for all the types because a lot of users want to use a treadmill, for example, as a bike. So lastCrankEventTime is actually used to simulate a cadence as a bike from a treadmill. So i would like to leave this as it is

Reply:
Acknowledged.